packages/elements/src/components/Layout/index.tsx

```diff
@@ -120,10 +150,38 @@ export const LevelItem: React.SFC<LayoutProps> = ({ children, className = '', da
   </div>
 )
 
-export const Section: React.SFC<LayoutProps> = ({ children, className = '', dataTest = '' }) => (
-  <div className={`section ${className}`} data-test={dataTest}>
+export const Section: React.SFC<SectionProps> = ({
+  isFlex,
+  isCentered,
+  isAlignedEnd,
+  isAlignedStart,
+  isFlexColumn,
+  children,
+  className = '',
+  dataTest = '',
+  hasPadding = true,
+  hasMargin = true,
+  hasBackground = true,
+  isFullHeight,
+}) => (
+  <section
+    className={cx(
+      'section',
+      className,
+      isFlex && 'is-flex',
+      isCentered && 'is-centered',
+      isAlignedEnd && 'is-aligned-end',
+      isAlignedStart && 'is-aligned-start',
+      isFlexColumn && 'is-flex-column',
+      hasPadding && 'has-padding',
+      hasMargin && 'has-margin',
+      hasBackground && 'has-background',
+      isFullHeight && 'is-full-height',
```

Comment:
I think `className` should come last so that we can override style with `className` props
https://github.com/JedWatson/classnames/issues/112